cmd/ig/daemon.go

```diff
@@ -65,6 +74,24 @@ func newDaemonCommand(runtime runtime.Runtime) *cobra.Command {
 		16384,
 		"The events buffer length. A low value could impact horizontal scaling.")
 
+	daemonCmd.PersistentFlags().StringVar(
+		&serverKey,
+		"tls-key",
+		"",
+		"Path to tls key file")
```

Comment:
Maybe now that we're also adding other keys directly to the params/flags or config file, we should rather name this `tls-key-file` _or_ accept the key contents directly. Same for the other params.